test/python/utest_masarclient/test_masarClient.py

```diff
@@ -0,0 +1,320 @@
+        self.assertEqual(result[0][1], 'masarExample0001')  # ChannelName 1
+        self.assertEqual(result[0][2], 'masarExampleBoUninit')  # ChannelName 2
+        self.assertEqual(result[0][3], 'masarExampleMbboUninit')  # ChannelName 3
+        self.assertEqual(result[0][4], 'masarExample0002')  # ChannelName 4
+        self.assertEqual(result[0][5], 'masarExample0003')  # ChannelName 5
+        self.assertEqual(result[0][6], 'masarExample0004')  # ChannelName 6
+        self.assertEqual(result[0][7], 'masarExampleCharArray')  # ChannelName 7
+        self.assertEqual(result[0][8], 'masarExampleShortArray')  # ChannelName 8
+        self.assertEqual(result[0][9], 'masarExampleLongArray')  # ChannelName 9
+        self.assertEqual(result[0][10], 'masarExampleStringArray')  # ChannelName 10
+        self.assertEqual(result[0][11], 'masarExampleFloatArray')  # ChannelName 11
+        self.assertEqual(result[0][12], 'masarExampleDoubleArray')  # ChannelName 12
+        self.assertEqual(result[0][13], 'masarExampleMbboUninitTest')  # ChannelName 13
+        self.assertEqual(result[1][0], 10)  # 0000 Value
+        self.assertEqual(result[1][1], 'string value')  # 0001 Value
+        self.assertEqual(result[1][2], '0')  # BoUninit Value (appropriately returned as string)
+        self.assertEqual(result[1][3], 1)  # MbboUninit Value
+        self.assertEqual(result[1][4], 'zero')  # 0002 Value
+        self.assertEqual(result[1][5], 'one')  # 0003 Value
+        self.assertEqual(result[1][6], 1.9)  # 0004 Value
+        self.assertEqual(result[1][7], ())  # CharArray Value
+        self.assertEqual(result[1][8], ())  # ShortArray Value
+        self.assertEqual(result[1][9], ())  # LongArray Value
+        self.assertEqual(result[1][10], ())  # StringArray Value
+        self.assertEqual(result[1][11], ())  # FloatArrayValue
+        self.assertEqual(result[1][12], ())  # DoubleArray Value
+        self.assertEqual(result[1][13], 1)  # MbboUninitTest Value
+        self.assertEqual(result[2], (5, 0, 0, 5, 0, 0, 6, 4, 1, 5, 0, 2, 6, 5))  # DBR type
+        self.assertEqual(result[3], (1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1))  # isConnected
+        self.assertEqual(result[4], (631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000, 631152000))  # SecondsPastEpoch
+        self.assertEqual(result[5], (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))  # NanoSeconds
+        self.assertEqual(result[6], (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))  # UserTag
+        self.assertEqual(result[7], (0, 0, 3, 0, 3, 0, 0, 3, 3, 3, 3, 3, 3, 0))  # Severity
+        self.assertEqual(result[8], (3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3))  # Status
+        self.assertEqual(result[9], ('UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM', 'UDF_ALARM'))  # Message
```

Comment:
This block of expected values looks like the previous one, will the first element removed.  Perhaps consolidate into a helper method?